src/listen.js

```diff
@@ -152,6 +152,23 @@ module.exports = function(defaultFuncs, api, ctx) {
                 })(0)
                 break;
               }
+              
+              if (v.delta.class == "ClientPayload") {
+                var clientPayload = utils.decodeClientPayload(v.delta.payload);
+                if (clientPayload && clientPayload.deltas) {
+                  for (var i in clientPayload.deltas) {
+                    var delta = clientPayload.deltas[i];
+                    if (delta.deltaMessageReaction) {
+                      delta.deltaMessageReaction.type = "message_reaction";
+                      delta.threadId = delta.threadKey.threadFbId;
```

Comment:
Oh, In my old parsed files, It's contains threadFbId, but today new parsed I have `otherUserFbId` too ! Maybe hotfix or something from Facebook side?